### PR TITLE
Fixing warnings breaking compilation on macOS

### PR DESCRIPTION
**Pull request by MartinSchmidtTuebingen**
Fixing warnings (wrong initialisation order, non-explicit conversion from Int_t to Double_t)

### CODE REVIEW

**Comment by dstocco** [comment]
+test

File: PWGPP/TPC/AliTPCcalibResidualPID.cxx [hunk 1]
```diff
@@ -203,7 +204,6 @@ AliTPCcalibResidualPID::AliTPCcalibResidualPID(const char *name)
     fTree_tanTheta_vs(0.),
     fTree_distance_nb(0.),
     fTree_distance_vs(0.),
```
Comment (dstocco) [inline PWGPP/TPC/AliTPCcalibResidualPID.cxx]: Hi @MartinSchmidtTuebingen ,
it seems that you forgot to remove a comma here.

Cheers,
Diego

**Comment by fprino** [comment]
+test

**Comment by fprino** [comment]
+1

**Pull request merged**